lib/Notes/utils/getMetadataFormResponse.js

```diff
@@ -2,10 +2,16 @@ const getMetadataPropsFromResponse = (responseMetadata) => {
   const {
     createdDate,
     createdByUsername: createdBy,
+    updatedByUsername,
     updatedDate: lastUpdatedDate,
-    updatedByUserId: lastUpdatedBy
+    updatedByUserId,
+    createdByUserId,
   } = responseMetadata;
 
+  const lastUpdatedBy = createdByUserId === updatedByUserId
```

Comment:
I think we can just use `updatedByUsername` in both cases.
If ids are equal - we're passing `createdBy`, which is the same as `updatedByUsername`
If they are not - we're passing `updatedByUsername`

So we can just remove the conditional